src/templates/lws-service_manifests.yaml.j2

```
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/name: {{ app_name }}-lws
+    control-plane: controller-manager
+  name: {{ app_name }}-lws-controller-manager-metrics-service
+  namespace: {{ namespace }}
+spec:
+  ports:
+  - name: https
+    port: 8443
+    protocol: TCP
+    targetPort: 8443
+  selector:
+    app.kubernetes.io/component: {{ app_name }}-lws-controller
+    control-plane: {{ app_name }}-controller-manager
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/component: webhook
+    app.kubernetes.io/created-by: lws
+    app.kubernetes.io/instance: webhook-service
+    app.kubernetes.io/name: {{ app_name }}-service
+    app.kubernetes.io/part-of: lws
+  name: {{ app_name }}-lws-webhook-service
+  namespace: {{ namespace }}
+spec:
+  ports:
+  - port: 443
+    protocol: TCP
+    targetPort: 9443
```

suggestion(non-blocking): Although I don't see much utility in this, let's use the context values of `{{ webhook_port }}` and `{{ webhook_target_port }}` for consistency, since we're doing the same thing in `jobset-service.yaml`
```suggestion
  - port: {{ webhook_port }}
    protocol: TCP
    targetPort: {{ webhook_target_port }}
```